### PR TITLE
Add detail sentence to CARS_template.md

### PR DESCRIPTION
**Pull request by maxeozakh**
**Description**

first step for #26956 
- patching `CARS_template.md` to add a detailed sentence to it

**Verification**

I generated new `CARS.md`, required for the next steps in #31109

### CODE REVIEW

File: selfdrive/car/CARS_template.md [hunk 1]
```diff
@@ -14,13 +14,17 @@ A supported vehicle is one that just works when you install a comma device. All
...
 |{{Column | map(attribute='value') | join('|') | replace(hardware_col_name, wide_hardware_col_name)}}|
 |---|---|---|{% for _ in range((Column | length) - 3) %}{{':---:|'}}{% endfor +%}
 {% for car_info in all_car_info %}
 |{% for column in Column %}{{car_info.get_column(column, star_icon, video_icon, footnote_tag)}}|{% endfor %}
-
+<!-- detail sentence:{{car_info.detail_sentence}} -->
 {% endfor %}
 
+<!-- ALL CAR INFO HERE ENDS -->
+
```
Comment (sshane) [inline selfdrive/car/CARS_template.md:27]:
```suggestion
<!-- ALL CAR INFO HERE ENDS -->

```

File: selfdrive/car/CARS_template.md [hunk 2]
```diff
@@ -14,13 +14,17 @@ A supported vehicle is one that just works when you install a comma device. All
 
 # {{all_car_info | length}} Supported Cars
 
+<!-- ALL CAR INFO HERE -->
+
```
Comment (sshane) [inline selfdrive/car/CARS_template.md:18]:
```suggestion
<!-- ALL CAR INFO HERE -->
```

Comment (maxeozakh) [reply selfdrive/car/CARS_template.md]: Done

File: selfdrive/car/CARS_template.md (hunk 1 above)
Comment (maxeozakh) [reply selfdrive/car/CARS_template.md]: Done

**Comment by maxeozakh** [comment]
Hey @sshane, is there anything else that needs to be done before we can merge this?

**Pull request closed without merging**